nativelib/src/main/scala/scala/scalanative/runtime/ExecutionContext.scala

```diff
@@ -25,4 +25,17 @@ object ExecutionContext {
       }
     }
   }
+
+  private[runtime] def loopRunOnce(): Int = {
```

Comment:
I understand & support your concerns for reasonably proper names.

I, for one, think the name is fine for this stage of the game, especially if the method
is marked Experimental.  

I think that a wide range of names are appropriate while we gain some experience.
The names here fall well within that range. It is certainly better than the `cycle()`
I came up with during rapid development of one of my similar design studies.